editor/engine_update_label.h

```diff
@@ -52,15 +52,14 @@ class EngineUpdateLabel : public LinkButton {
 	enum class VersionType {
 		STABLE,
 		RC,
-		BETA,
 		ALPHA,
+		BETA,
```

Comment:
Why this change? This would consider an alpha version as newer than a beta one

Reply:
🤦‍♂️